example/default.yaml

```diff
@@ -113,6 +113,8 @@ keybinds:
           key: [Ctrl: 'b',]
         - action: [Quit,]
           key: [Ctrl: 'q',]
+        - action: [DumpScreen,]
+          key: [Char: '/']
```

Comment:
I'm a little on the fence about putting this in the (default) config at the moment